.github/workflows/test.yml

```diff
@@ -0,0 +1,65 @@
+name: test
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+      - 'v[0-9]*'
+    tags:
+      - 'v*'
+  pull_request:
+    paths-ignore:
+      - '.github/releases.json'
+      - 'README.md'
+      - 'docs/**'
+
+env:
+  SETUP_BUILDX_VERSION: "latest"
+  SETUP_BUILDKIT_IMAGE: "moby/buildkit:latest"
+  TESTFLAGS: "-v --parallel=6 --timeout=30m"
+  TESTFLAGS_DOCKER: "-v --parallel=1 --timeout=30m"
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        worker:
+          - docker
+          - docker-container
+          - remote
+        pkg:
+          - ./tests
+        typ:
+          - integration
+        include:
+          - pkg: ./...
+            skip-integration-tests: 1
```

Comment:
I think we can remove this for now. We are already running unit tests in the build workflow.